src/Xamarin.Android.Sdk/targets/Xamarin.Android.Sdk.DefaultItems.props

```diff
@@ -29,4 +29,13 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
```

Comment:
We should remove this explicit reference ItemGroup now that the `FrameworkReference` below is in place.

Let's move this in to `Xamarin.Android.Sdk.Lite.props` (e.g. https://github.com/xamarin/xamarin-android/blob/net5-preview/src/Xamarin.Android.Sdk/targets/Xamarin.Android.Sdk.Lite.props#L17)